src/dataflow/source/kafka.rs

```diff
@@ -531,7 +530,11 @@ fn downgrade_capability(
 /// when the message queue switches from nonempty to empty.
 struct GlueConsumerContext(Mutex<SyncActivator>);
 
-impl ClientContext for GlueConsumerContext {}
+impl ClientContext for GlueConsumerContext {
+    fn stats(&self, statistics: Statistics) {
+        info!("Client stats: {:#?}", statistics);
+    }
```

Comment:
Any particular reason to override the default implementation here?

Reply:
{:#} instead of {}